pkg/runc/Dockerfile

```diff
@@ -11,7 +11,7 @@ RUN \
   make \
   && true
 ENV GOPATH=/go PATH=$PATH:/go/bin
-ENV RUNC_COMMIT=v1.0.0-rc4
+ENV RUNC_COMMIT=0351df1c5a66838d0c392b4ac4cf9450de844e2d
```

Comment:
Thanks, I'd been meaning to do this since I forgot to do it when I did the ctrd bump (and then continued to forget)

But is this the correct version?
```
$ git grep github.com/opencontainers/runc v1.0.0-beta.1 -- vendor.conf
v1.0.0-beta.1:vendor.conf:github.com/opencontainers/runc 593914b8bd5448a93f7c3e4902a03408b6d5c0ce
$ git grep RUNC v1.0.0-beta.1  -- RUNC.md
v1.0.0-beta.1:RUNC.md:RUNC_COMMIT = 593914b8bd5448a93f7c3e4902a03408b6d5c0ce
```

Ah, I see this is the version from containerd master. I expect beta.2 is not too far away and will incorporate this, shall we (by which I mean I) just do the bump of both when that happens?

Reply:
I hear that beta3 may be today